outputs.tf

```diff
@@ -1,6 +1,6 @@
 output "ssh_bastion_command" {
```

Comment:
Should this be an output of the EKS module? The idea was to keep the default username in one place

Reply:
I believe it should be an output of the bastion itself, moved there